ci/builder/Dockerfile

```diff
@@ -202,7 +260,7 @@ RUN npx pyright@$(sh /workdir/pyright-version.sh) --help
 
 COPY requirements.txt /workdir/
 
-RUN pip3 install -r /workdir/requirements.txt && rm /workdir/requirements*.txt
+RUN pip3 install --break-system-packages -r /workdir/requirements.txt && rm /workdir/requirements*.txt
```

Comment:
?!

Reply:
As of [Bookworm and onwards](https://salsa.debian.org/python-team/packages/python-pip/-/blob/master/debian/NEWS) `pip` will no longer install outside of a virtual environment without this argument. As we're building a Docker image for short-lived instantiations here it seems relatively manageable to globally install dependencies over the package manager's objections.

Reply:
Ah, ok, it's Ubuntu's fault. Ok, seems fine.